plugins/modules/gitlab_label.py

```diff
@@ -268,13 +268,18 @@ def update_label(self, var_obj):
             return True, True
         _label = self.gitlab_object.labels.get(var_obj.get('name'))
 
+        if (var_obj.get('new_name') is None) and (var_obj.get('color') is None):
+            self._module.fail_json(msg="Failed to update label, new_name or color must be present: %s " % _label)
```

Comment:
This might be better suited as a `required_one_of` clause for the `labels` parameter.

Adding something like:
```
required_one_of=[('new_name', 'color')]
```
after line 425 below. Beware that this might have other impacts, so take this with a grain of salt.

Reply:
Hi @russoz , I know about `required_one_of`, I tought about it, but it's not the case to use it because those field are not always required:
- new_name is a field useful only for label's editing
- color is requested in more payload, but not for all

Reply:
Perfect, thanks for confirming.